ribs/emitters/opt/_cma_es.py

```diff
@@ -146,7 +146,8 @@ def check_stop(self, ranking_values):
         Args:
             ranking_values (np.ndarray): Array of objective values of the
                 solutions, sorted in the same order that the solutions were
-                sorted when passed to tell().
+                sorted when passed to ``tell()``.
+
         Returns:
             True if any of the stopping conditions are satisfied.
         """
```

Comment:
Can you add a comment on line 163 about why we use `norm`? It has to do with how we may have multiple ranking values.